tests_integration/test_data/projects/snowpark_v2/snowflake.yml

```diff
@@ -0,0 +1,35 @@
+definition_version: 2
+
+defaults:
+  stage: "dev_deployment"
+
+entities:
+  hello_procedure:
+    type: "procedure"
+    identifier:
+      name: "hello_procedure"
+    handler: "a.hello_procedure"
+    signature:
+      - name: "name"
+        type: "string"
+    returns: string
+    artifacts:
+      - "app_1/"
+
+  test:
+    type: "procedure"
```

Comment:
Let's change stage for one of these to test both cases (two object on single stage + objects on multiple stages)